syntax = "proto3";
package v1;

option java_package = "com.protoconf.agent.api.v1";

import "google/protobuf/any.proto";

message ConfigSubscriptionRequest {
    string path = 1;
}

message ConfigUpdate {
    google.protobuf.Any value = 1;
}

service ProtoconfService{
    rpc SubscribeForConfig(ConfigSubscriptionRequest) returns (stream ConfigUpdate);
}
